app/v1/controllers/requestParsers/validators/AmendOtherDeductionsValidator.scala

```diff
@@ -16,13 +16,17 @@
 
 package v1.controllers.requestParsers.validators
 
+import config.AppConfig
+import javax.inject.Inject
+import utils.CurrentTaxYear
 import v1.controllers.requestParsers.validators.validations._
 import v1.models.errors.{MtdError, RuleIncorrectOrEmptyBodyError}
 import v1.models.request.amendOtherDeductions.{AmendOtherDeductionsBody, AmendOtherDeductionsRawData, Seafarers}
 
-class AmendOtherDeductionsValidator extends Validator[AmendOtherDeductionsRawData] {
+class AmendOtherDeductionsValidator @Inject()(implicit appConfig: AppConfig, currentTaxYear: CurrentTaxYear)
```

Comment:
Coverage seems low for the validator

Reply:
The coverage is low with the body format validation, I didn't touch that. And I'm not sure how I would raise it.